### PR TITLE
Improve developer experience

### PR DESCRIPTION
**Pull request by posix4e**
This pull request was originally a code pull request which I thought might make people's life a bit easier as documentation.  It also removes broken links from the readme.

    // FREEBIE

### CODE REVIEW

**Comment by posix4e** [comment]
@moxie0 Good call on no need for the shell script. Maven really is mostly self explainable. I think it's nice to have an agreed upon environment which isn't heroku. If heroku is the official environment I'm happy to switch the instructions to that. If nothing else, stating that you need java 8 will avoid other issues in the future.

File: README.md [hunk 1]
```diff
@@ -31,6 +31,32 @@ Current BitHub Payment Per Commit:
 ![Current Price](https://bithub.herokuapp.com/v1/status/payment/commit)
 
 
+This Repository
+-------------
+This signal server is a dropwizard java application built with maven and deployed
+wit a heroku procfile.
```
Comment (odeke-em) [inline README.md]: s/wit /with /g

File: README.md [hunk 2]
```diff
@@ -47,3 +73,4 @@ License
 Copyright 2013-2016 Open Whisper Systems
 
 Licensed under the AGPLv3: https://www.gnu.org/licenses/agpl-3.0.html
+
```
Comment (odeke-em) [inline README.md]: A spurious new line here.

File: README.md [hunk 3]
```diff
@@ -31,6 +31,32 @@ Current BitHub Payment Per Commit:
 ![Current Price](https://bithub.herokuapp.com/v1/status/payment/commit)
 
 
+This Repository
+-------------
+This signal server is a dropwizard java application built with maven and deployed
```
Comment (odeke-em) [inline README.md]: deployed to Heroku with a Procfile.

Comment (odeke-em) [inline README.md]: Signal Server is a

**Comment by posix4e** [comment]
Wow can't believe i made those typos. Hopefully got your suggestions

**Comment by posix4e** [comment]
Going to close this commit before it goes stale , it's now in https://github.com/WhisperSystems/Signal-Server/pull/81

**Comment by maincui** [comment]
Hi, @posix4e , is this a topic about Signal-Server installation guide from start, step by step? I found no developer-friendly means to setup the server as a private one, it's badly needed.

**Comment by posix4e** [comment]
Sorry @maincui I have stopped doing any work with signal server. I am not skilled enough to help them in any way

**Comment by maincui** [comment]
@posix4e , thank you anyway for notifying me of your current status, I will try joining another team to contribute as they have made it, leaving some documenting and finalizing work to to be done. I think it's a good chance to catch up by learning while helping.

**Pull request closed without merging**